readthedocs/oauth/models.py

```diff
@@ -168,8 +169,17 @@ class RemoteRepository(models.Model):
         null=True,
         blank=True,
     )
-
     json = models.TextField(_('Serialized API response'))
+    # TODO: Make remote_id and vcs_provider not nullable and
+    #  unique together after migration
```

Comment:
Would be good to create an issue or a note card in the project to not forget about this.

Reply:
Done! :)